sdk/spring/CHANGELOG.md

```diff
@@ -1,49 +1,151 @@
 # Release History
 
 ## 4.0.0-beta.4 (Unreleased)
-- Support Spring Boot version: 2.5.5+ and 2.6.0 - 2.6.3.
-- Support Spring Cloud version: 2020.0.3+ and 2021.0.0.
+- Support Spring Boot version: 2.5.5+ and 2.6.0+.
+- Support Spring Cloud version: 2020.0.3+ and 2021.0.0+.
 
 ### Dependency Updates
 - Upgrade dependency according to spring-boot-dependencies:2.6.3 and spring-cloud-dependencies:2021.0.0.
 
+### Features Added 
+- Add `Automatic-Module-Name` for all Spring Cloud Azure modules and change the root pacakge names to match the module names [#27350](https://github.com/Azure/azure-sdk-for-java/issues/27350), [#27420](https://github.com/Azure/azure-sdk-for-java/pull/27420).
+
+### Spring Cloud Azure Dependencies (BOM)
+#### Dependency Updates
+- Use `azure-sdk-bom:1.2.0` instead of including each sdk dependencies [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
+- Use `azure-cosmos:4.28.0` instead of the version `4.27.0` in `azure-sdk-bom` [#27850](https://github.com/Azure/azure-sdk-for-java/pull/27850).
+- Use `azure-resourcemanager:2.13.0`.
+- Use `azure-spring-data-cosmos:3.19.0`.
+
 ### Spring Cloud Azure Starter Active Directory
 This section includes changes in `spring-cloud-azure-starter-active-directory` module.
 
 #### Breaking Changes
-- Remove the AAD conditional access filter. [#27727](https://github.com/Azure/azure-sdk-for-java/pull/27727)
+- Delete the AAD conditional access filter. [#27727](https://github.com/Azure/azure-sdk-for-java/pull/27727)
+- Rename classes and methods [#27273](https://github.com/Azure/azure-sdk-for-java/pull/27273), [#27579](https://github.com/Azure/azure-sdk-for-java/pull/27579):
+  + Rename classes from `AAD/AADB2C` to `Aad/AadB2c`.
+  + Rename method in `UserPrincipal` from `getKid` to `getKeyId`.
+  + Rename methods in `AADAuthenticationProperties` from `allowedGroupIdsConfigured/allowedGroupNamesConfigured` to `isAllowedGroupIdsConfigured/isAllowedGroupNamesConfigured`.
+  + Rename methods in `AADAuthorizationServerEndpoints` from `authorizationEndpoint/endSessionEndpoint/jwkSetEndpoint/tokenEndpoint` to `getAuthorizationEndpoint/getEndSessionEndpoint/getJwkSetEndpoint/getTokenEndpoint`.
+  + Rename method in `UserGroupProperties` from `getUseTransitiveMembers` to `isUseTransitiveMembers`.
+- Delete `AadJwtClaimValidator` and `AadJwtAudienceValidator` [#27365](https://github.com/Azure/azure-sdk-for-java/pull/27365):
+  + Delete `AadJwtClaimValidator` and use `JwtClaimValidator` instead.
+  + Delete `AadJwtAudienceValidator` and use `JwtClaimValidator` instead.
+  + Rename `AadTokenClaim` to `AadJwtClaimNames`.
```

Comment:
Is this suitable here?
<img width="421" alt="image" src="https://user-images.githubusercontent.com/4465723/160036441-d5039026-cda2-48b4-a732-c97a2cd221a9.png">

Reply:
good point